docker.py

```diff
@@ -21,24 +21,30 @@
     ('centos7_py34-pyenv', '3.4.7')]
 
 base_choices = [a[0] for a in _base_choices]
-cuda_choices = ['none', 'cuda70', 'cuda75', 'cuda80', 'cuda90']
+cuda_choices = ['none', 'cuda70', 'cuda75', 'cuda80', 'cuda90', 'cuda91']
 cudnn_choices = [
     'none', 'cudnn4', 'cudnn5', 'cudnn5-cuda8', 'cudnn51',
-    'cudnn51-cuda8', 'cudnn6', 'cudnn6-cuda8', 'cudnn7-cuda8', 'cudnn7-cuda9']
-nccl_choices = ['none', 'nccl1.3.4', 'nccl2.0-cuda8', 'nccl2.0-cuda9']
+    'cudnn51-cuda8', 'cudnn6', 'cudnn6-cuda8', 'cudnn7-cuda8', 'cudnn7-cuda9',
+    'cudnn7-cuda91']
+nccl_choices = [
+    'none', 'nccl1.3.4', 'nccl2.0-cuda8', 'nccl2.0-cuda9',
+    'nccl2.1.4-cuda91']
 
 cuda_cudnns = {
     'cuda70': ['cudnn4'],
     'cuda75': ['cudnn4', 'cudnn5', 'cudnn51', 'cudnn6'],
     'cuda80': ['cudnn5-cuda8', 'cudnn51-cuda8', 'cudnn6-cuda8',
                'cudnn7-cuda8'],
     'cuda90': ['cudnn7-cuda9'],
+    'cuda91': ['cudnn7-cuda91'],
 }
 cuda_nccls = {
     'cuda70': ['nccl1.3.4'],
     'cuda75': ['nccl1.3.4'],
     'cuda80': ['nccl1.3.4', 'nccl2.0-cuda8'],
-    'cuda90': ['nccl2.0-cuda9'],  # CUDA 9 does not support nccl 1.3
+    # CUDA 9 does not support nccl 1.3
+    'cuda90': ['nccl2.0-cuda9'],
+    'cuda91': ['nccl2.1.4-cuda91'],
```

Comment:
I agree that `nccl2.1-cuda91` is better.
Just to make sure, NCCL 1.x is using `nccl1.3.4`. Is this because 1.3.4 is the final version?
https://github.com/kmaehashi/chainer-test/blob/150f778e7125e83c4e058fad6d65b569758f0ba7/docker.py#L41-L48

Reply:
It is because we didn't have a policy about nccl version that time. I think `nccl1.3` is better.

Reply:
OK, I fixed that too.